source/support_contact_and_more_information/index.html.md.erb

```diff
@@ -9,45 +9,32 @@ weight: 220
 
 You can get notifications about GOV.UK Pay’s status to your email, phone, RSS reader or website, by subscribing on our [status page](https://payments.statuspage.io/#).
 
-## Incident severity
+After you contact us, the GOV.UK Pay team will contact you using the Zendesk ticketing system.
 
-GOV.UK Pay incident severities are defined as follows:
+## Report an incident
```

Comment:
should we tell people where to find the emergency contact info? it's in the email we send when a service went live (and it's in some of the MoUs).

Reply:
Did you see I added it further down the page?

Reply:
oh sorry yes missed this!